shared/chat/conversation/info-panel/bot.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react'
+import * as Kb from '../../../common-adapters'
+import * as Styles from '../../../styles'
+import {FeaturedBot} from 'constants/types/rpc-gen'
+
+type Props = FeaturedBot & {
+  description?: string
+  onShowProfile: (username: string) => void
+}
+
+const Bot = ({botAlias, description, botUsername, ownerTeam, ownerUser, onShowProfile}: Props) => {
+  const lower = (
+    <Kb.Box2 direction="horizontal" alignItems="center" gap="xtiny" alignSelf="flex-start">
+      {description !== '' && (
+        <Kb.Text type="BodySmall" lineClamp={1}>
+          {description}
+        </Kb.Text>
+      )}
+    </Kb.Box2>
+  )
+
+  const usernameDisplay = (
+    <Kb.Box2 direction="horizontal" alignSelf="flex-start">
+      <Kb.Text type="BodySmallSemibold" style={{color: Styles.globalColors.black}}>
+        {botAlias || botUsername}
+      </Kb.Text>
+      <Kb.Text type="BodySmall">&nbsp;• by @{ownerTeam ?? ownerUser ?? botUsername}</Kb.Text>
```

Comment:
this shouldn't default to `botUsername`, one of `ownerTeam` or `ownerUser` will always be set. I think this is supposed to be a link to the team/user page and not just text

Reply:
I defaulted to botUsername in cases where there's a restricted bot that isn't in our database (like dr jenkins for example). adding a link shouldn't be too tricky

Reply:
ah nice makes sense

Reply:
Getting the team to show a popup is a larger process than expected, so I'll throw that into another ticket